app/src/download.js

```diff
@@ -208,11 +208,14 @@
 			if ( !thisName || !thisDependencies ) {
 				return;
 			}
-			thisDependencies = thisDependencies.split( "," );
+			thisDependencies = thisDependencies.split( "," ).map(function( dependency ) {
+				// remove any path prefixes, ain't caring about those
+				return dependency.replace( /^.+\//, '' );
+			});
 			dependencies[ thisName ] = $();
 			$.each( thisDependencies, function() {
 				var dependecy = this,
-					dependecyElem = $( "[name=" + this + "]" );
+					dependecyElem = $( ".components-area input[type=checkbox][name='" + this + "']" );
```

Comment:
This ended up selecting the version radio input, instead of the "version" module (which we don't show).